destroy-cluster/go.mod

```diff
@@ -0,0 +1,57 @@
+module example/user/destroy-cluster
+
+go 1.17
+
+require (
+	github.com/IBM-Cloud/bluemix-go v0.0.0-20211006063826-ee9901463fa3
+	github.com/IBM/go-sdk-core/v4 v4.10.0
+	github.com/IBM/go-sdk-core/v5 v5.7.0
+	github.com/IBM/networking-go-sdk v0.24.0
+	github.com/IBM/platform-services-go-sdk v0.21.0
+	github.com/IBM/vpc-go-sdk v1.0.1
+	github.com/ppc64le-cloud/pvsadm v0.1.3
```

Comment:
As I mentioned earlier, `github.com/ppc64le-cloud/pvsadm` repo is not meant for exporting outside the project, let's not depend on this repo here, there is no guarantee we can give for the clients(they may change frequently)

Reply:
Removed